src/org/opensolaris/opengrok/index/Indexer.java

```diff
@@ -206,11 +193,24 @@ public static void main(String argv[]) {
             // Assemble the unprocessed command line arguments (possibly
             // a list of paths). This will be used to perform more fine
             // grained checking in invalidateRepositories().
-            for (int optind=0; optind< argv.length; optind++) {
+            for (int optind = 0; optind < argv.length; optind++) {
```

Comment:
Maybe use enhanced-for loop?

Reply:
I'd leave it for now - it's a code I did not change (other than whitespace).